mlflow/pyfunc/loaders/chat_model.py

```diff
@@ -74,7 +84,14 @@ def predict(
             Model predictions in :py:class:`~ChatCompletionResponse` format.
         """
         messages, params = self._convert_input(model_input)
-        response = self.chat_model.predict(self.context, messages, params)
+        parameters = inspect.signature(self.chat_model.predict).parameters
+        if parameters.get("context") or len(parameters) == 3:
+            _logger.info(
+                _DROP_CONTEXT_IN_CHAT_MODEL_PREDICT_INFO.replace("<FUNCTION_NAME>", "predict")
```

Comment:
why not use `_DROP_CONTEXT_IN_CHAT_MODEL_PREDICT_INFO.format`?

Reply:
```python
>>> "{a} {a} {a}".format(a="x")
'x x x'
```

Reply:
Ah :)